test/utils/run_test_coverage.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+set -exo pipefail
+CDIR="$(cd "$(dirname "$0")/.." ; pwd -P)"
+VERBOSITY=2
+
+export TORCH_TEST_DEVICES="$CDIR/pytorch_test_base.py"
+export CPU_NUM_DEVICES=4
+
+function run_test {
+  coverage run -p "$@"
```

Comment:
I think we need to direct coverage to the `--source` here

Reply:
https://stackoverflow.com/questions/67980243/coverage-py-only-shows-test-files

Reply:
Thanks for poiting this out, pointing it to `torch_xla/` let's see what it does.

Reply:
It didn't produce something meaningful (0% coverage), reverting -- let's investigate different types of coverage in the next round.